app/models/playbook.rb

```diff
@@ -4,17 +4,22 @@
 #
 # Table name: playbooks
 #
-#  id          :bigint           not null, primary key
-#  name        :string
-#  description :string
-#  created_at  :datetime         not null
-#  updated_at  :datetime         not null
+#  id             :bigint           not null, primary key
+#  name           :string
+#  available_gear :int
+#  description    :string
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
 #
 # The class or playbook that the hunter has
 # provides unique abilities to the Hunter
 class Playbook < ApplicationRecord
   has_many :improvements
-  has_many :gears
+  has_many :playbook_gears
```

Comment:
I think our dependent option should be `destroy`.

Reply:
Indeed.  Resolved with commit: 977bbfe